src/codeFiles.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
This can be removed now (even with static import, there's no reason to merge this file)